src/Commands.js

```diff
@@ -37,6 +38,7 @@
             fired = [],
             prefix = "cmdId_",
             indexer = 0,
+            rethrow, //default true
```

Comment:
I totally agree about the need for a way to pass a logging method via the initialization options.
This requirement is in our roadmap and will be implemented soon.